### ISSUE TITLE
VSC Extension does not start/connect xtext server

### ISSUE THREAD

**Issue by TobiasNx** [metafacture/metafacture-fix#315]
![grafik](https://github.com/metafacture/metafacture-fix/assets/61879957/bea5766e-8900-49c7-89a4-f357096c948a)

Only syntax highlighting seems to work.

**Comment by TobiasNx** [comment]
![grafik](https://github.com/metafacture/metafacture-fix/assets/61879957/2f873cee-bc96-4bdd-9606-f1eafa8eca88)

**Comment by katauber** [comment]
functional review: @TobiasNx 
code review: @fsteeg

**Comment by TobiasNx** [comment]
@katauber maybe an interesting observation: the Flux exstenstion seems to launch with no problem.

**Comment by katauber** [comment]
We found out that the Xtext Server in the extension is not working on Linux when the extensuib is build in Windows. When build in the Linux subsystem of Windows, the extension works fine on Linux.
@TobiasNx already tested the behaviour. I just published the new build extension on https://open-vsx.org/. I also tried to publish the new version on https://marketplace.visualstudio.com/ but I got the same problem like the last time ("Upload error"). I'll try again and again until it works.
